force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +232,20 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
+        String resultString = '';
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
+        
+        for (Integer i = 1; i < repeatCount; i++) {
 
-            // If it is not the last iteration, add a semicolon to separate the strings
+            resultString = resultString + inputString + '; ';// If it is not the last iteration, add a semicolon to separate the strings
```

Comment:
Use the [addition assignment operator](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_expressions_operators_understanding.htm#:~:text=Addition%20assignment%20operator)